test-data/unit/fine-grained-follow-imports.test

```diff
@@ -638,6 +638,7 @@ import p2.m2
 p1/m1.py:1: error: "int" not callable
 main.py:2: error: Cannot find implementation or library stub for module named "p2.m2"
 main.py:2: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
+main.py:2: error: Cannot find implementation or library stub for module named "p2"
```

Comment:
This now happens because the flags have `--namespace-packages` for this test! Before, p2 was being found as a namespace package because it contained `p2/m2.py.2`. This sort of error is happening all over the place in #9636